include/boost/geometry/algorithms/detail/signed_index_type.hpp

```diff
@@ -21,6 +21,7 @@ namespace boost { namespace geometry
 
 
 typedef boost::make_signed<std::size_t>::type signed_index_type;
+typedef boost::make_signed<std::size_t>::type signed_size_type;
```

Comment:
Thanks!
The filename should be renamed too.
I don't understand that I only see buffer-things, if this renaming was part of this PR, then I would expect much more changes? Where are they?